plugins/module_utils/_connection_options.py

```diff
@@ -108,7 +108,7 @@ def _filter(k, v):
 
         retry_action = hvopts.pop('retry_action')
         if 'retries' in hvopts:
-            hvopts['session'] = self._get_custom_requests_session(new_callback=self._retry_callback_generator(retry_action), **hvopts.pop('retries'))
+            hvopts['session'] = self._get_custom_requests_session(new_callback=self._retry_callback_generator(retry_action), conopt_verify=self._conopt_verify, **hvopts.pop('retries'))
```

Comment:
I'm debating whether we should bother passing this as parameter. Instead, we can make the original call here, and then set `hvopts['session'].verify = self._conopt_verify`. It feels like that may be the correct place to do this. Let me know if you object.

Reply:
We can do it the way you suggested. I actually had it that way in my first draft, but then I thought it wouldn't match the rest of the style of the code and changed it.
Should I change it?